.git-blame-ignore-revs

```diff
@@ -4,3 +4,6 @@
 
 # Format all files with Prettier
 2f3b359987bebf00d45a8c32d58e3b302d117c4a
+
+# Format element Python files with black
+a0f42e54dea00d363b2a7e443d6085d5fd3923ea
```

Comment:
Will switch this out once we decide which files to run this on / before this is merged.